<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Occlusion</class>
 <widget class="QWidget" name="Occlusion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Occlusion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Occlusion</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="AmbientRadiusLabel">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>AmbientRadius</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="AmbientRadiusSlider">
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #bbb;
    background: white;
    height: 10px;
    border-radius: 5px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #eee, stop:1 #ccc);
    border: 1px solid #777;
    width: 20px;
    margin: -5px 0;
    border-radius: 10px;
}
QSlider::sub-page:horizontal {
    background: rgb(123, 248, 255);
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="AmbientBias">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>AmbientBias</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="AmbientBiasSlider">
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #bbb;
    background: white;
    height: 10px;
    border-radius: 5px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #eee, stop:1 #ccc);
    border: 1px solid #777;
    width: 20px;
    margin: -5px 0;
    border-radius: 10px;
}
QSlider::sub-page:horizontal {
    background: rgb(123, 248, 255);
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AmbientLernelLabel">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>AmbientKernel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="AmbientLernelSlider">
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #bbb;
    background: white;
    height: 10px;
    border-radius: 5px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #eee, stop:1 #ccc);
    border: 1px solid #777;
    width: 20px;
    margin: -5px 0;
    border-radius: 10px;
}
QSlider::sub-page:horizontal {
    background: rgb(123, 248, 255);
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="AmbientBlur">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>AmbientBlur</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="OnpushButton">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
   color:white;
   background-color: rgb(0, 0, 0);
    border-bottom:5px solid #626569;

   border-radius:10px;
}
QPushButton:hover
{
   color:white;
   background-color: #1f75d9;
    border-bottom:5px solid #165195;
   border-radius:10px;
}
</string>
             </property>
             <property name="text">
              <string>On</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="OffpushButton">
             <property name="font">
              <font>
               <family>Arial Black</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
   color:white;
   background-color: rgb(0, 0, 0);
    border-bottom:5px solid #626569;

   border-radius:10px;
}

QPushButton:hover
{
   color:white;
   background-color: rgb(255, 44, 107);
    border-bottom:5px solid rgb(255, 14, 50);
   border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string>Off</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>